invoker/core/src/main/java/com/google/cloud/functions/invoker/BackgroundFunctionExecutor.java

```diff
@@ -250,27 +278,33 @@ void serviceCloudEvent(CloudEvent cloudEvent) throws Exception {
         throw new IllegalStateException("Event has no \"data\" component");
       }
     }
+  }
+
+  private static class CloudEventFunctionExecutor extends FunctionExecutor<Void>{
```

Comment:
Heh, I've been struggling with my IDE a bit because I can't get it to accept that shift-space is the same as space, and not something it should ignore.